RELNOTES

```diff
@@ -1,5 +1,10 @@
 firejail (0.9.68rc2) baseline; urgency=low
   * work in progress
+  * security: the firejail package on Ubuntu was left vulnerable to
```

Comment:
i'm wondering if this should really go to the release notes. it's nothing specific to this release. and users obtaining the software via source or their distro probably don't care about that.

Reply:
I had the same concern we reading it.

Reply:
@reinerh [on Feb 1](https://github.com/netblue30/firejail/pull/4889#discussion_r797060998):

> i'm wondering if this should really go to the release notes. it's nothing
> specific to this release. and users obtaining the software via source or
> their distro probably don't care about that.

The CVE seems like an important issue and I couldn't think of a better way to
try to make users aware of how security is handled in the package vs in the
PPA.  Users that already installed the distro package might just keep using it
without even knowing that the PPA exists.  If some users know that and don't
care as you say that's fine; but for someone to decide whether they care about
something or not, they need to be aware of it in the first place.

This information is currently only on README.md, which is something that I
wouldn't expect anyone to check periodically or to even read more than once.

Personally, for the projects I'm interested in, I usually don't go around
checking the README/issues/discussions on every release, but I might read the
release notes/announcement, which is where I would hope to find at least the
breaking changes.  And I think it would be nice to have important suggestions
in there as well.

> it's nothing specific to this release.

Well, it's not a code change, but the PPA recommendation was added to README.md
in this release.  I think of this entry as a documentation change + a
security-related (and packaging-related) announcement.

Reply:
Is the issue mostly that the entry is verbose?

If so, what if it was reduced to:

```
  * security: on Ubuntu, the PPA is now recommended over the distro package
    (see README.md) (#4748)
```

Reply:
I like your suggested wording much better. The old one sounds (in my opinion) a bit too accusing.

Reply:
@reinerh [on Feb 2](https://github.com/netblue30/firejail/pull/4889#discussion_r797932049):

> I like your suggested wording much better. The old one sounds (in my opinion)
> a bit too accusing.

Yes; I just wanted to make the entry self-contained (that is, not vague) and I
had attempted to simplify and write it in different ways, but all of them also
ended up sounding bad, so I just went with the original matter-of-fact way.

Anyway, changed to the new text.